src/Components/Publishing/Sections/FullscreenViewer/FullscreenViewer.tsx

```diff
@@ -68,7 +68,7 @@ export class FullscreenViewer extends React.Component<FullscreenViewerProps, Ful
   render() {
     const sliderSettings = {
       dots: false,
-      infinite: true,
+      infinite: false,
```

Comment:
Per offline discussion with Owen -- the scroll-to-beginning-from-end didn't match rest of site